scripts/windows/build/Create-ArmBase.ps1

```diff
@@ -57,7 +57,7 @@ foreach($ImageTag in $DockerImageTags)
 {
     Write-Host "Docker build image tag [$ImageTag]"
     
-    $docker_build_cmd = "docker build --no-cache"
+    $docker_build_cmd = "docker -H npipe:////./pipe/iotedge_moby_engine build --no-cache"
```

Comment:
It depends how to set up your device to build base image.  Since you documented down how to set up the device for building base image.  It is ok to have this option.  Please put a comment to explain why have this option here.